components/badge/badge.css

```diff
@@ -0,0 +1,93 @@
+:root {
+    --white-color: white;
+    --online-color: green;
+    --offline-color: lightslategray;
+    --busy-color: red;
+}
+
+.badge {
+    border-radius: 50%;
+    position: absolute;
+    border: 1px solid var(--white-color);
+}
+
+.badge-online {
+    background-color: var(--online-color);
+}
+
+.badge-offline {
+    background-color: var(--offline-color);
+}
+
+.badge-busy {
+    background-color: var(--busy-color);
+}
+
+.badge-size-xxl {
+    width: 1.5rem;
+    height: 1.5rem;
+    font-size: 12px;
+}
+
+.badge-size-xl {
+    width: 1.4rem;
+    height: 1.4rem;
+    font-size: 14px;
+}
+
+.badge-size-lg {
+    width: 1.2rem;
+    height: 1.2rem;
+}
+
+.badge-size-md {
+    width: 1rem;
+    height: 1rem;
+}
+
+.badge-size-sm {
+    width: 10px;
+    height: 10px;
+}
+
+.badge-position-lg {
+    bottom: 4px;
+    right: 12px;
```

Comment:
use rem here for 'right' just like you used rems everywhere. Otherwise, the code looks good for me to merge

Reply:
someone told me to use px units when the measurement is less than 1rem. should I use rems even it is less than 1rem? 
Asking this to know which is better practice.